DOCS.md

```diff
@@ -1,20 +1,34 @@
-Use this plugin for sending build status notifications via Email. You can
```

Comment:
Please take drone-slack or drone-docker as an example for this file. You are not mentioning how to set the secrets.